src/main/java/org/sonarlint/intellij/editor/SonarExternalAnnotator.java

```diff
@@ -63,13 +63,15 @@ public void apply(@NotNull PsiFile file, AnnotationContext annotationResult, @No
     }
 
     Collection<LiveIssue> issues = annotationResult.store.getForFile(file.getVirtualFile());
-    for (LiveIssue issue : issues) {
-      // reject non-null ranges that are no longer valid. It probably means that they were deleted from the file.
-      RangeMarker range = issue.getRange();
-      if (range == null || range.isValid()) {
-        addAnnotation(issue, holder);
-      }
-    }
+    issues.stream()
+      .filter(issue -> !issue.isResolved())
+      .forEach((issue) -> {
```

Comment:
![MINOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-minor.png) Remove the parentheses around the "issue" parameter [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](https://dory.sonarsource.com/sonarqube/coding_rules#rule_key=squid%3AS1611)